docroot/themes/humsci/humsci_basic/templates/html.html.twig

```diff
@@ -68,10 +68,9 @@
     {% set classes = classes|merge(['role--' ~ role|clean_class]) %}
   {% endfor %}
   <body{{ attributes.addClass(classes) }}>
-      <a href="#main-content" class="visually-hidden focusable su-skipnav">
-        {{ 'Skip to main content'|t }}
-      </a>
-    {#}{ pattern("brandbar", {}, brand_bar_variant) }#}
+    <a href="#main-content" class="visually-hidden focusable su-skipnav">
```

Comment:
hmm should this be higher up on the page? in the masthead perhaps?

Reply:
we could put this in page.html.twig instead

Reply:
This is going to need some attention style wise so we might want to wait until we have a card for it. I was just fixing the indenting.

Reply:
ah ok @kaseybon yeah if you wouldnt mind making a card, or maybe it could go on the focus styles bug card we made too? https://sparkbox.atlassian.net/browse/STN-160